src/store/model/galaxus.ts

```diff
@@ -8,7 +8,7 @@ export const Galaxus: Store = {
 			text: ['In den Warenkorb']
 		},
 		maxPrice: {
-			container: '.productDetail .Z1ex',
+			container: '.productDetail .Z1f2',
```

Comment:
Alright, so after this being dynamic, I think we need another way to solve this as the updated selector no longer works either.

![image](https://user-images.githubusercontent.com/12074633/103912916-e5e24700-50d5-11eb-935c-437eb3b75859.png)

Let's see if we can come up with something else.

It seems that using just `.productDetail` works here. Do you want to test and update?

Thanks!

Reply:
Using only ".productDetail"  works most parts, since i have some special trackings on the galaxus site aswell, i found that as soon that a number comes before the acutal price, the script takes that number (even if it s an percentage)

Example: https://www.galaxus.de/de/s1/product/lian-li-o11dynamic-xl-midi-tower-pc-gehaeuse-12200197
![image](https://user-images.githubusercontent.com/449003/103926439-362ec880-5119-11eb-8343-7ee03b6cde6d.png)
![image](https://user-images.githubusercontent.com/449003/103926543-5b233b80-5119-11eb-8557-27c978f32a9b.png)
![image](https://user-images.githubusercontent.com/449003/103926583-670efd80-5119-11eb-9605-c438eced9885.png)
![image](https://user-images.githubusercontent.com/449003/103926740-9e7daa00-5119-11eb-80d2-6988f4f3f20b.png)

So it looks like it works until there is running something like a discount or something like this ( in this case )

Reply:
> i found that as soon that a number comes before the acutal price, the script takes that number (even if it s an percentage)

Well that sucks 😞

Also, updated selector again:

![image](https://user-images.githubusercontent.com/12074633/104033243-47b9b400-519d-11eb-85b9-8512628382ec.png)

Reply:
Already updated like 10 minutes ago in bugfix branch, should be up to date now ( until the next change i think on sunday / monday ) ...